examples/crm/src/activity/ActivityLogIterator.tsx

```diff
@@ -0,0 +1,83 @@
+import { Button, List } from '@mui/material';
+import { useState } from 'react';
+import {
+    COMPANY_CREATED,
+    CONTACT_CREATED,
+    CONTACT_NOTE_CREATED,
+    DEAL_CREATED,
+    DEAL_NOTE_CREATED,
+} from '../consts';
+import { Activity } from '../types';
+import { ActivityLogCompanyCreated } from './ActivityLogCompanyCreated';
+import { ActivityLogContactCreated } from './ActivityLogContactCreated';
+import { ActivityLogContactNoteCreated } from './ActivityLogContactNoteCreated';
+import { ActivityLogDealCreated } from './ActivityLogDealCreated';
+import { ActivityLogDealNoteCreated } from './ActivityLogDealNoteCreated';
+
+const PAGE_SIZE = 20;
+
+type ActivityLogIteratorProps = {
+    activities: Activity[];
+};
+
+export function ActivityLogIterator({ activities }: ActivityLogIteratorProps) {
+    const [activitiesDisplayed, setActivityDisplayed] = useState(PAGE_SIZE);
+
+    const filteredActivities = activities.slice(0, activitiesDisplayed);
+
+    return (
+        <List
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
**note (non-blocking):** ‏I Think we need more spacing between activities

Reply:
Validated by RA team